src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -346,18 +346,32 @@ export function transform(formConfig, form) {
     ),
   );
 
-  // Transform the related disabilities lists into an array of strings
   if (clonedData.vaTreatmentFacilities) {
-    const newVAFacilities = clonedData.vaTreatmentFacilities.map(facility =>
-      _.set(
+    const newVAFacilities = clonedData.vaTreatmentFacilities.map(facility => {
+      // Transform the related disabilities lists into an array of strings
+      const newFacility = _.set(
         'treatedDisabilityNames',
         transformRelatedDisabilities(
           facility.treatedDisabilityNames,
           claimedConditions,
         ),
         facility,
-      ),
-    );
+      );
+
+      // If the day is missing, set it to the last day of the month because EVSS requires a day
```

Comment:
This whole chunk could be abstracted into a function and shared between MVP and all claims. Needs some tweaking for that to work, though.